src/vivarium_gates_iv_iron/components/hemoglobin.py

```diff
@@ -16,6 +21,70 @@ def __init__(self):
     def name(self):
         return "hemoglobin"
 
+    def setup(self, builder: Builder):
+        self.randomness = builder.randomness.get_stream(self.name)
+        self.columns_created = ["country", "hemoglobin_distribution_propensity", "hemoglobin_percentile"]
+        # load data
+        mean = builder.data.load(data_keys.HEMOGLOBIN.MEAN)
+        stddev = builder.data.load(data_keys.HEMOGLOBIN.STANDARD_DEVIATION)
+        # TODO replace --->>>
+        location_weights = mean.copy().drop(columns=["value"])
+        # lookup data, categorical column "country"
+        mean_dfs, stddev_dfs = [], []
+        # TODO have this in artifact and correct the location weights
+        for country in ["Bangladesh", "Pakistan", "India"]:
+            loc_mean = mean.copy()
+            loc_stddev = stddev.copy()
+            loc_mean["country"] = country
+            loc_stddev["country"] = country
+            mean_dfs.append(loc_mean)
+            stddev_dfs.append(loc_stddev)
+            location_weights[country] = 1/3  # will be pop of country over pop of region
+        index_columns = ["country", "sex", "age_start", 'age_end', 'year_start', 'year_end']
+        mean = pd.concat(mean_dfs).set_index(index_columns)["value"].rename("mean")
+        stddev = pd.concat(stddev_dfs).set_index(index_columns)["value"].rename("stddev")
+        # <<<--- TODO replace
+        distribution_parameters = pd.concat([mean, stddev], axis=1).reset_index()
+        # TODO: look in Risk code to make sure we have canonical exposure naming
```

Comment:
You should address these canonical naming todos before merging.

Reply:
Fair point!

Reply:
Corrected with commit.